From c3f92d667cacf3a8236400362e3b3a170a23ccdc Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Tue, 15 Jan 2019 12:30:30 +0100
Subject: [PATCH] power: pmic-uclass: allow selection of CONFIG_PMIC_CHILDREN
 via config header

temporary workaround to enable devicetree setup in u-boot for other I2C drivers

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 drivers/power/pmic/pmic-uclass.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/power/pmic/pmic-uclass.c b/drivers/power/pmic/pmic-uclass.c
index 9347b40688..903a630d6b 100644
--- a/drivers/power/pmic/pmic-uclass.c
+++ b/drivers/power/pmic/pmic-uclass.c
@@ -18,7 +18,7 @@
 
 DECLARE_GLOBAL_DATA_PTR;
 
-#if CONFIG_IS_ENABLED(PMIC_CHILDREN)
+#ifdef CONFIG_PMIC_CHILDREN
 int pmic_bind_children(struct udevice *pmic, ofnode parent,
 		       const struct pmic_child_info *child_info)
 {
